custom_components/tuya_local/devices/dr_heater_dr008_thermostat.yaml

```diff
@@ -0,0 +1,229 @@
+name: Dr. Heater DR-008 Electric Radiant Floor Thermostat
+primary_entity:
+  entity: climate
+  dps:
+    - id: 101
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: "Smart"
+              value: auto
+            - dps_val: "Manual"
+              value: heat
+            - dps_val: "Anti_frozen"
```

Comment:
These are all heat modes, so I don't think mapping them incorrectly to hvac_mode is the right thing to do. 103 can just be preset_mode, if translation_key is set to thermostat, "manual", "program" and "away" are available as presets.

Reply:
True, this is an electric floor heater, so it only provides heat. I can change this if you think that is preferrable?

Reply:
Yes, I think hvac_mode just off and heat, and using preset_mode to switch between the modes is preferable.